src/components/Layout/Header.tsx

```diff
@@ -110,159 +229,143 @@ const Header: FunctionComponent<Props> = props => {
+                                    <li className="align-items-center nav-item" role="presentation">
+                                        <a
+                                            className="nav-link"
+                                            href="https://sourcegraph.com/search"
+                                            data-button-style={buttonStyle.text}
+                                            data-button-location={buttonLocation.nav}
+                                            data-button-type="cta"
+                                        >
                                             Search code
                                         </a>
                                     </li>
                                 )}
-                                <li className="header__nav-item nav-item" role="presentation">
+                                <li className="align-items-center nav-item" role="presentation">
                                     <Link href="/demo" passHref={true}>
                                         {/* eslint-disable-next-line jsx-a11y/anchor-is-valid */}
-                                        <a className="nav-link">Request a demo</a>
+                                        <a
+                                            className="nav-link"
+                                            data-button-style={buttonStyle.text}
+                                            data-button-location={buttonLocation.nav}
+                                            data-button-type="cta"
+                                        >
+                                            Request a demo
+                                        </a>
                                     </Link>
                                 </li>
                                 {!props.hideGetStartedButton && (
-                                    <li className="header__nav-item nav-item" role="presentation">
+                                    <li className="align-items-center nav-item" role="presentation">
                                         <Link href="/get-started" passHref={true}>
-                                            <a href="#none" className="nav-link">
+                                            {/* eslint-disable-next-line jsx-a11y/anchor-is-valid */}
+                                            <a
+                                                className="nav-link"
+                                                data-button-style={buttonStyle.text}
+                                                data-button-location={buttonLocation.nav}
+                                                data-button-type="cta"
+                                            >
                                                 Get started
                                             </a>
                                         </Link>
```

Comment:
We could even reduce the markup here as well by creating a `rightNavLinks` array